product-service/src/services/pg_postProduct.ts

```diff
@@ -0,0 +1,83 @@
+import { PostSneaker } from 'src/types/types';
+import { Client, QueryResultRow } from 'pg';
+import { dbConfig } from '@app/libs/dbConfig';
+import { postProductSQL } from '@app/sql';
+import { logger } from '@app/utils/logger';
+
+const productErrorText = 'Product was not added to database!';
+
+export const PG_postProduct = async ({
```

Comment:
it's better use https://www.npmjs.com/package/validator or https://joi.dev for validation. Or create a schema for body